src/remote/server/ReplServer.cpp

```diff
@@ -75,8 +75,24 @@ namespace
 	const USHORT CTL_VERSION1 = 1;
 	const USHORT CTL_CURRENT_VERSION = CTL_VERSION1;
 
-	volatile bool* shutdownPtr = NULL;
+	volatile bool shutdownFlag = false;
```

Comment:
Not the single problematic place we have, but actually this is not correct.
Solution with `std::call_once` would be safe.

Reply:
@asfernandes  In this particular case - no danger. shutdownHandler() may be called only from fb_shutdown(), which is reliably synchronized internally, i.e. 2 calls in parallel never happens.